tf_encrypted/keras/layers/activation_test.py

```diff
@@ -0,0 +1,24 @@
+# pylint: disable=missing-docstring
+import unittest
+
+import tensorflow as tf
+
+import tf_encrypted as tfe
+from tf_encrypted.keras.testing_utils import agreement_test
+
+class TestActivation(unittest.TestCase):
+
+  def setUp(self):
+    tf.reset_default_graph()
+
+  def test_activation_relu(self):
+    self._core_activation(activation="relu")
+
+  def _core_activation(self, **layer_kwargs):
```

Comment:
can't you inline this function?

Reply:
yes, but this follows the more general construction of testing demonstrated in dense_test.py and convolutional_test.py, which seems preferable when considered in context